example-config.yml

```diff
@@ -460,6 +460,7 @@ itinerary:
 #           0.9: 'Mostly Accessible'
 #       popups:
 #         testTarget: Test Target!
+#         testTarget-narrow: Target! # used for narrow UI elements such as link buttons under trip details.
```

Comment:
We should probably specify that this is optional!

Reply:
Updated in https://github.com/opentripplanner/otp-react-redux/pull/729/commits/866fdc2bfbe16ae5c0c97c0868f3bd3a6f37e541.